petab/core.py

```diff
@@ -399,14 +401,17 @@ def get_optimization_to_simulation_parameter_mapping(
         The sbml model with observables and noise specified according to the
         petab format. Optional if par_sim_ids is provided.
 
-    par_opt_ids, par_sim_ids: list of str, optional
+    par_sim_ids: list of str, optional
         Ids of the optimization and simulation parameters. If not passed,
```

Comment:
Opt not needed any more? Adapt description

Reply:
Was unused.